packages/react-core/src/components/Button/Button.tsx

```diff
@@ -79,7 +79,7 @@ export interface ButtonProps extends Omit<React.HTMLProps<HTMLButtonElement>, 'r
   /** Forwarded ref */
   innerRef?: React.Ref<any>;
   /** Adds count number to button */
-  badgeProps?: BadgeCountProps;
+  badgeVariant?: BadgeCountObject;
```

Comment:
Nit.  So usually when we use "variant", we are switching between different variations of something.  lFor instance the [Button](https://www.patternfly.org/v4/components/button#variant-examples) has a variant prop that switches the styling of the button.  I would suggest `countOptions` for a prop name here.